core/src/executor/evaluate/mod.rs

```diff
@@ -211,6 +211,14 @@ async fn evaluate_function<'a, T>(
 
     match func {
         // --- text ---
+        Function::Concat(exprs) => {
+            let mut vec: Vec<Evaluated<'_>> = vec![];
+            for expr in exprs {
+                vec.push(eval(expr).await?);
+            }
+
+            f::concat(name(), vec) // _s).await?),
```

Comment:
Unnecessary comment is in here, it would be good to remove it.

Reply:
+1